### DISCUSSION TITLE
Triggers are being called twice

### DISCUSSION THREAD

**Discussion by AntonBSystance** [microsoft/Dynamics365Commerce.Solutions#313] [Q&A]
Hi everyone,

Would appreciate if anyone can give a hint for D365 Store Commerce extension development in Pos. 

Currently, I have an issue, that the extension to the trigger is being triggered twice, independently from the trigger type (I tried PostOpenShift and PostOperation triggers). From the development console there is just a sign of the extension triggered two times from the pos core.

If I put the simple log, I see from the dev console that the trigger is called two times (Screen attached). Is there any reason why trigger extension can be called twice?

Console:
![image](https://github.com/user-attachments/assets/1188376a-a2bb-41ea-a632-82bd500b93e1)

Manifest:
![image](https://github.com/user-attachments/assets/fcfa1a0e-fe59-4b2b-95f7-9313b6edf5a0)

Code in PostOpenShift:
![image](https://github.com/user-attachments/assets/b8c5f21c-69bc-4739-86b7-52e37c6e82dd)

Thanks in advance for any help on the matter :)

**Comment by dsdharmen91** [comment]
this is because you have two triggers. one is shift open and other is operation. Operation will execute every new operation either shift open or logon or item search. you can reach to dsdharmen91@gmail.com
